### PR TITLE
fix: modify getActiveKeys() to return list of previous keys

### PR DESCRIPTION
**Pull request by mattstrom**
RKT-7833

### CODE REVIEW

**Comment by mattstrom** [comment]
I've had trouble deploying a version for `javascript-sdk`, so getting the next semver for the SDK is holding up this PR.

**Comment by FrazerBayley** [comment]
hotfix-v1.3

File: src/index.js [hunk 1]
```diff
@@ -25,18 +26,20 @@ function main(client, options, _performance) {
...
 				return;
 			}
-			
+
 			const timeNow = Date.now();
 			const domContentLoadedEventStart = (_performance || performance).timing.domContentLoadedEventStart;
 			var threshold = options.timeoutThreshold || 60000;
 			if (domContentLoadedEventStart === 0 || timeNow < domContentLoadedEventStart + threshold) {
-				setTimeout(invokeFunctions, MAX_TIMEOUT);
+				setTimeout(function() {
```
Comment (FrazerBayley) [inline src/index.js:40]: Did you change this? I though we werent passing the subset for this?

Comment (FrazerBayley) [reply src/index.js:40]: I thought you chose not to give the subset to the timeout

Comment (mattstrom) [reply src/index.js:40]: I did change it after a discussion with Rob. We opted to let all of the timeouts get their chance. One of the changes we made later on ended up handling the behavior anyway.

Comment (FrazerBayley) [reply src/index.js:40]: Oh ok

**Pull request merged**